jest.config.ts

```diff
@@ -1,6 +1,7 @@
 const config = {
   preset: 'ts-jest',
   testEnvironment: 'node',
+  modulePathIgnorePatterns: ['<rootDir>/dart-sass/'],
```

Comment:
I had a similar but different commit to fix this issue: https://github.com/sass/embedded-host-node/pull/327/commits/01d7e42d3ae79d2413c61c1a359ad654524ffccd

Not sure which way is preferred.

Reply:
Yeah, your solution looks better. I'll just add it to this PR for simplicity